common/api/core-common.api.md

```diff
@@ -9245,7 +9245,7 @@ export class TextString {
     widthFactor?: number;
 }
 
-// @public
+// @beta

```

Comment:
this is breaking

Reply:
Not breaking, this was just added, and it's only used by the ElementGeometry api which is all beta...

Reply:
Added in #6291, merged 3 days ago. Not yet in any release.